README.rst

```diff
@@ -0,0 +1,77 @@
+========
+Workflow
+========
+
+This is the MIT Libraries' `Airflow <https://airflow.apache.org/>`_ implementation. **Please read these instructions carefully, because this is very much a shared codebase.**
+
+.. contents:: Table of Contents
+.. section-numbering::
+
+Intro
+-----
+
+This repo contains a few different things. The main attraction is the ``workflows`` directory which contains all our Airflow DAGs. There is also the Docker infrastructure used for deploying, both locally for development and in production. Finally, there's a custom Python command line tool for performing various maintenace tasks on the Fargate cluster.
```

Comment:
Consider linking the first usage of `DAG` to http://airflow.apache.org/concepts.html#dags